Hunted Mobile/Hunted Mobile/Hunted Mobile/Repository/GameRepository.cs

```diff
@@ -10,15 +10,20 @@
 namespace Hunted_Mobile.Repository {
     public class GameRepository {
         public async Task<Game> GetGame(int gameId) {
-            var response = new HttpClientResponse();
+            HttpClientResponse response = new HttpClientResponse();
             await response.Convert(HttpClientRequestService.Get($"games/{gameId}"));
 
+            string[] location = response.Item.GetValue("police_station_location").ToString().Split(',');
             return new Game() {
                 Id = response.GetNumberValue("id"),
                 Duration = response.GetNumberValue("duration"),
                 Interval = response.GetNumberValue("interval"),
                 TimeLeft = response.GetNumberValue("time_left"),
                 Status = response.GetStringValue("status"),
+                PoliceStationLocation = new Location() {
+                    Latitude = double.Parse(location[0]),
+                    Longitude = double.Parse(location[1])
+                },
```

Comment:
Als het goed is zou dit ook moeten werken:
```
                PoliceStationLocation = new Location(
                    response.GetStringValue("police_station_location")
                )
```
Die constructor van Location doet dan het parsen.